bindings/node/package.json

```diff
@@ -36,44 +38,24 @@
   "dependencies": {
     "bindings": "^1.5.0",
     "node-addon-api": "^4.3.0",
-    "prebuild-install": "^7.1.1",
-    "socks": "^2.7.1"
+    "prebuild-install": "^7.1.1"
   },
   "devDependencies": {
-    "@types/sinon": "^10.0.14",
-    "@types/sinon-chai": "^3.2.9",
+    "@types/bindings": "^1.5.1",
+    "@typescript-eslint/eslint-plugin": "^6.0.0",
     "chai": "^4.3.7",
```

Comment:
Given there's nothing left in test/ can we also remove chai, chai-subset, and mocha?